util/util.go

```diff
@@ -3,12 +3,19 @@ package util
 import (
 	"os"
 	"path/filepath"
+	"regexp"
 	"runtime"
 
 	"github.com/inconshreveable/log15"
 	"golang.org/x/xerrors"
 )
 
+// CveIDPattern is regexp matches to `CVE-\d{4}-\d{4,}`
+var CveIDPattern = regexp.MustCompile(`CVE-\d{4}-\d{4,}`)
+
+// IncorrectCveIDPattern is regexp matches regexp `CVE-\S+` to detects incorrect IDs by comparing the number of matches with CVEIDPattern. 
+var IncorrectCveIDPattern = regexp.MustCompile(`CVE-\S+`)
```

Comment:
Can't I just use `strings.Count(ref.Title, "CVE-")`?
https://pkg.go.dev/strings#Count

Reply:
Since it can take CVE-IDs that are not incorrect, I thought it might be a bit strange in terms of naming conventions (if we were naming them, we could use strictCveIDPattern/looseCveIDPattern?